src/pages/TodoList/TodoList.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+import { Component } from "react";
+import Button from "../../ui-kit/components/Button/Button";
+import Input from "../../ui-kit/components/Input/Input";
+import styles from "./TodoList.module.scss"
+class TodoList extends Component {
+
+    state = {
+        name: "",
+        items: [],
+        rerender: true
+    }
+
+    componentDidMount() {
+        const itemsArr = JSON.parse(localStorage.getItem("items"))
+        if (itemsArr) {
+            this.setState({ items: itemsArr })
+        }
+    }
+    onChange = (event) => {
+        const { value } = event.target
+        this.setState({ name: value })
+    }
+
+    onSubmit = (event) => {
+        const { name, items, rerender } = this.state
+        event.preventDefault()
+        items.push({ name: name })
+        localStorage.setItem("items", JSON.stringify(items))
+        this.setState({ rerender: !rerender })
+
+    }
+    deleteItem = (index) => {
+        const { items, rerender } = this.state
+        items.splice(index, 1)
+        localStorage.setItem("items", JSON.stringify(items))
```

Comment:
const { items, rerender } = this.state;
const newItems = [...items];
newItems .splice(index, 1)
this.setState({ items: newItems })